core/shared/src/main/scala/zio/internal/FiberRuntime.scala

```diff
@@ -308,40 +311,28 @@ final class FiberRuntime[E, A](fiberId: FiberId.Runtime, fiberRefs0: FiberRefs,
 
         case FiberMessage.YieldNow =>
 
-        case message =>
```

Comment:
I'm not sure what this case was meant for since all the extensions of `FiberMessage` are included above 🤔
I removed it but let me know if you think we should add it back for some reason

Reply:
It was probably legacy from a different time.